features/storage/csi_resize.feature

```diff
@@ -75,3 +75,86 @@ Feature: CSI Resizing related feature
     Examples:
       | sc_name |
       | gp2-csi | # @case_id OCP-25809
+
+  # @author ropatil@redhat.com
+  @admin
+  Scenario Outline: [CSI Resize] offline volume expansion from 1Gi to 2Gi
+    Given the master version >= "4.8"
```

Comment:
Azure Disk CSI driver is not installed by default until 4.10, so I think we could let it run after 4.10(using polarion version), and remove this line.

Reply:
Yeah sure. Any possibility of adding other platform if they supporting this feature below 4.10? 
If no will change this to 4.10.

Reply:
I have no idea, I prefer to using polarion parameter when it is "Scenario Outline" which need consider other cases.

Reply:
We could add tags to the examples if each of the examples supports a different version. See example here - https://github.com/openshift/cucushift/pull/8709/files

Please also add a `@4.10` tag. This tag will be useful for executing test scenarios in Prow.